src/components/calculator/ActivityRiskControls.tsx

```diff
@@ -64,19 +64,27 @@ export const ActivityRiskControls: React.FunctionComponent<{
         source={Interaction}
       />
       <SelectControl
-        id="setting"
-        label="Ventilation"
+        id="distance"
+        label="Distance (most of the time)"
         data={data}
         setter={setter}
-        source={Setting}
+        source={Distance}
       />
       <SelectControl
-        id="distance"
-        label="Distance (most of the time)"
+        id="setting"
+        label="Ventilation"
         data={data}
         setter={setter}
-        source={Distance}
+        source={Setting}
       />
+      {data.setting === 'outdoor' && data.distance === 'intimate' ? (
+        <div className="warning">
+          Due to the very close distances and exchange of bodily fluid, we are
+          not confident that being outdoors reduces the risk in a substantial
+          way. Thus, we are not providing any bonus for being outdoors when
+          intimate.
```

Comment:
Since this is doing double duty as "cuddling" and "kissing/having sex" I wonder if it should say something like: 

> Due to the very close distances and **possible** exchange of bodily fluids, we are...

Reply:
Good catch. done.